notebooks/dc2/catalog_generation/matchingLensGalaxies.ipynb

```diff
@@ -0,0 +1,6293 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import GCRCatalogs\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import pickle\n",
+    "import matplotlib.pyplot as plt\n",
+    "from astropy.io import fits\n",
+    "from GCR import GCRQuery"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load Data from GCR"
```

Comment:
Could we please include a markdown cell with a short summary, just with the title and a brief description (e.g. assigning cosmoDC2 lens galaxies to the lensing systems in OM10 and Goldstein et al 2019)?

Reply:
Added.